FushaarProvider/src/main/kotlin/com/fushaar/FushaarProvider.kt

```diff
@@ -0,0 +1,135 @@
+            "Sci-fi" to "$mainUrl/gerne/sci-fi",
+            "Crime" to "$mainUrl/gerne/crime",
+            "Drama" to "$mainUrl/gerne/drama",
+            "Adventure" to "$mainUrl/gerne/adventure",
+            "Biography" to "$mainUrl/gerne/biography",
+            "Music" to "$mainUrl/gerne/music",
+            "Sport" to "$mainUrl/gerne/sport",
+            "Documentary" to "$mainUrl/gerne/documentary",
+            "History" to "$mainUrl/gerne/history",
+            "Family" to "$mainUrl/gerne/family",
+            "Romance" to "$mainUrl/gerne/romance",
+            "Mystery" to "$mainUrl/gerne/mystery"
+        )
+        val pages = moviesUrl.apmap {
+            val doc = app.get(it.second).document
+            val list = doc.select("article.poster").mapNotNull { element ->
+                element.toSearchResponse()
+            }
+            HomePageList(it.first, list)
+        }.sortedBy { it.name }
+        return HomePageResponse(pages)
+    }
+
+    override suspend fun search(query: String): List<SearchResponse> {
+        val q = query.replace(" ", "%20")
+        val result = arrayListOf<SearchResponse>()
+        listOf(
+            "$mainUrl/?s=$q",
+        ).apmap { url ->
+            val d = app.get(url).document
+            d.select("article.poster").mapNotNull {
+                it.toSearchResponse()?.let { it1 -> result.add(it1) }
+            }
+        }
+        return result.distinct().sortedBy { it.name }
+    }
+
+    override suspend fun load(url: String): LoadResponse {
+        var doc = app.get(url).document
+        val posterUrl = doc.select("figure.poster noscript img").attr("src")
+        val year = doc.select("header span.yearz").text()?.getIntFromText()
+        val title = doc.select("header h1").text()+"\n"+doc.select("header h2").text()
+        val synopsis = doc.select("div.postz").text()
+        val trailer = doc.select("div.rll-youtube-player iframe").attr("src")
+        val tags = doc.select("div.z-info li").map { it.text() }
+
+
+        return newMovieLoadResponse(
+            title,
+            url,
+            TvType.Movie,
+            "$url"
+        ) {
+            this.posterUrl = posterUrl.also{print("posterurl2 :"+it)}
+            this.year = year
+            this.tags = tags
+            this.plot = synopsis
+            addTrailer(trailer)
+        }
+    }
+
+
+    override suspend fun loadLinks(
+        data: String,
+        isCasting: Boolean,
+        subtitleCallback: (SubtitleFile) -> Unit,
+        callback: (ExtractorLink) -> Unit
+    ): Boolean {
+        app.get(data).document
+            .select("#fancyboxID-8 > script").map {
+                callback.invoke(
+                    ExtractorLink(
+                        source = this.name,
+                        name = name,
+                        url = it.html().substring(252,384),
+                        referer = this.mainUrl,
+                        quality = 1080,

```

Comment:
change 1080 to [Qualities](https://recloudstream.github.io/dokka/app/com.lagradost.cloudstream3.utils/-qualities/index.html).[Unknown](https://recloudstream.github.io/dokka/app/com.lagradost.cloudstream3.utils/-qualities/-unknown/index.html).[value](https://recloudstream.github.io/dokka/app/com.lagradost.cloudstream3.utils/-qualities/-unknown/value.html)